lib/vanagon/platform/windows.rb

```diff
@@ -119,8 +119,9 @@ def generate_msi_package(project)
         #   -gg             - Generate GUIDS now
         #   -dr             - Directory reference to root directories (cannot contains spaces e.g. -dr MyAppDirRef)
         #   -sreg           - Suppress registry harvesting.
-        #   -var <variable> - Substitute File/@Source="SourceDir" with a preprocessor or a wix variable
-        "cd $(tempdir); \"$$WIX/bin/heat.exe\" dir staging -v -ke -indent 2 -cg #{cg_name} -gg -dr #{dir_ref} -t wix/#{project.name}.filter.xslt -sreg -var var.StageDir -out wix/#{project.name}-harvest.wxs",
+        "cd $(tempdir); \"$$WIX/bin/heat.exe\" dir staging -v -ke -indent 2 -cg #{cg_name} -gg -dr #{dir_ref} -t wix/#{project.name}.filter.xslt -sreg -out wix/#{project.name}-harvest.wxs",
+        # candle individual files to generate the wxo files.
+        "cd $(tempdir); \"$$WIX/bin/candle.exe\"  -arch #{@architecture} -out wix/#{project.name}-harvest.wixobj   wix/#{project.name}-harvest.wxs  ",
```

Comment:
@melissa Have taken the command directly from the Rakefile so need to determine exactly what options are actually necessary. Same applies to the WixUtil and UI extension options.

Yes - was thinking either of a glob, or specifying the filenames as a list so we can generalise the command at least in the Ruby code (even if we have to express it out as individual Makefile commands).

The "candle" command is not well formed yet - need to work on that.

Think we do need the .out.